ERROR - 2025-05-08 19:40:55 --> Email: sendWithSmtp throwed ErrorException: stream_socket_enable_crypto(): SSL operation failed with code 1. OpenSSL Error messages:
error:0A00010B:SSL routines::wrong version number in C:\wamp64\www\sites\Projeto1\Back-end\system\Email\Email.php:1917
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'stream_socket_e...', 'C:\\wamp64\\www\\s...', 1917)
#1 C:\wamp64\www\sites\Projeto1\Back-end\system\Email\Email.php(1917): stream_socket_enable_crypto(Resource id #11, true, 121)
#2 C:\wamp64\www\sites\Projeto1\Back-end\system\Email\Email.php(1810): CodeIgniter\Email\Email->SMTPConnect()
#3 C:\wamp64\www\sites\Projeto1\Back-end\system\Email\Email.php(1695): CodeIgniter\Email\Email->sendWithSmtp()
#4 C:\wamp64\www\sites\Projeto1\Back-end\system\Email\Email.php(1587): CodeIgniter\Email\Email->spoolEmail()
#5 C:\wamp64\www\sites\Projeto1\Back-end\app\Controllers\EmailController.php(88): CodeIgniter\Email\Email->send()
#6 C:\wamp64\www\sites\Projeto1\Back-end\app\Controllers\Cliente.php(81): App\Controllers\EmailController->verificaEmail(Array)
#7 C:\wamp64\www\sites\Projeto1\Back-end\system\CodeIgniter.php(941): App\Controllers\Cliente->create()
#8 C:\wamp64\www\sites\Projeto1\Back-end\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
#9 C:\wamp64\www\sites\Projeto1\Back-end\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\wamp64\www\sites\Projeto1\Back-end\public\index.php(85): CodeIgniter\CodeIgniter->run()
#11 {main}
ERROR - 2025-05-08 19:40:55 --> Email: Unable to send email using SMTP. Your server might not be configured to send mail using this method.
ERROR - 2025-05-08 19:40:55 --> 
hello: 
The following SMTP error was encountered: 
starttls: 220 proxy.email-ssl.com.br ESMTP Postfix

ERROR - 2025-05-08 19:41:30 --> mysqli_sql_exception: Duplicate entry '588.888.888-88' for key 'cliente.CPF' in C:\wamp64\www\sites\Projeto1\Back-end\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `cl...', 0)
#1 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `cl...')
#2 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `cl...')
#3 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `cl...', Array, false)
#4 C:\wamp64\www\sites\Projeto1\Back-end\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\wamp64\www\sites\Projeto1\Back-end\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#6 C:\wamp64\www\sites\Projeto1\Back-end\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\wamp64\www\sites\Projeto1\Back-end\app\Controllers\Cliente.php(55): CodeIgniter\Model->insert(Array)
#8 C:\wamp64\www\sites\Projeto1\Back-end\system\CodeIgniter.php(941): App\Controllers\Cliente->create()
#9 C:\wamp64\www\sites\Projeto1\Back-end\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
#10 C:\wamp64\www\sites\Projeto1\Back-end\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\wamp64\www\sites\Projeto1\Back-end\public\index.php(85): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2025-05-08 19:41:30 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '588.888.888-88' for key 'cliente.CPF'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `cliente` (`nome`, `CPF`, `RG`, `telefone`, `email`, `endereco`, `datanascimento`, `treino_com_personal`, `personal_id`, `atestado_medico`, `termo_responsabilidade`, `status`) VALUES (:nome:, :CPF:, :RG:, :telefone:, :email:, :endereco:, :datanascimento:, :treino_com_personal:, :personal_id:, :atestado_medico:, :termo_responsabilidade:, :status:)', [...], false)
 2 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Cliente.php(55): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Cliente->create()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-05-08 19:41:30 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '588.888.888-88' for key 'cliente.CPF'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `cliente` (`nome`, `CPF`, `RG`, `telefone`, `email`, `endereco`, `datanascimento`, `treino_com_personal`, `personal_id`, `atestado_medico`, `termo_responsabilidade`, `status`) VALUES (\'breno teste\', \'588.888.888-88\', \'45.847.444-4\', \'(12) 99656-9274\', \'breno@sitenivel.com.br\', \'Av. Macedo Soares, Campos do Jordão\', \'2004-06-07\', \'nao\', \'null\', NULL, NULL, \'inativo\')')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `cliente` (`nome`, `CPF`, `RG`, `telefone`, `email`, `endereco`, `datanascimento`, `treino_com_personal`, `personal_id`, `atestado_medico`, `termo_responsabilidade`, `status`) VALUES (\'breno teste\', \'588.888.888-88\', \'45.847.444-4\', \'(12) 99656-9274\', \'breno@sitenivel.com.br\', \'Av. Macedo Soares, Campos do Jordão\', \'2004-06-07\', \'nao\', \'null\', NULL, NULL, \'inativo\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `cliente` (`nome`, `CPF`, `RG`, `telefone`, `email`, `endereco`, `datanascimento`, `treino_com_personal`, `personal_id`, `atestado_medico`, `termo_responsabilidade`, `status`) VALUES (:nome:, :CPF:, :RG:, :telefone:, :email:, :endereco:, :datanascimento:, :treino_com_personal:, :personal_id:, :atestado_medico:, :termo_responsabilidade:, :status:)', [...], false)
 4 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\Cliente.php(55): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Cliente->create()
 9 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-05-08 19:41:30 --> [Caused by] mysqli_sql_exception: Duplicate entry '588.888.888-88' for key 'cliente.CPF'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `cliente` (`nome`, `CPF`, `RG`, `telefone`, `email`, `endereco`, `datanascimento`, `treino_com_personal`, `personal_id`, `atestado_medico`, `termo_responsabilidade`, `status`) VALUES (\'breno teste\', \'588.888.888-88\', \'45.847.444-4\', \'(12) 99656-9274\', \'breno@sitenivel.com.br\', \'Av. Macedo Soares, Campos do Jordão\', \'2004-06-07\', \'nao\', \'null\', NULL, NULL, \'inativo\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `cliente` (`nome`, `CPF`, `RG`, `telefone`, `email`, `endereco`, `datanascimento`, `treino_com_personal`, `personal_id`, `atestado_medico`, `termo_responsabilidade`, `status`) VALUES (\'breno teste\', \'588.888.888-88\', \'45.847.444-4\', \'(12) 99656-9274\', \'breno@sitenivel.com.br\', \'Av. Macedo Soares, Campos do Jordão\', \'2004-06-07\', \'nao\', \'null\', NULL, NULL, \'inativo\')')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `cliente` (`nome`, `CPF`, `RG`, `telefone`, `email`, `endereco`, `datanascimento`, `treino_com_personal`, `personal_id`, `atestado_medico`, `termo_responsabilidade`, `status`) VALUES (\'breno teste\', \'588.888.888-88\', \'45.847.444-4\', \'(12) 99656-9274\', \'breno@sitenivel.com.br\', \'Av. Macedo Soares, Campos do Jordão\', \'2004-06-07\', \'nao\', \'null\', NULL, NULL, \'inativo\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `cliente` (`nome`, `CPF`, `RG`, `telefone`, `email`, `endereco`, `datanascimento`, `treino_com_personal`, `personal_id`, `atestado_medico`, `termo_responsabilidade`, `status`) VALUES (:nome:, :CPF:, :RG:, :telefone:, :email:, :endereco:, :datanascimento:, :treino_com_personal:, :personal_id:, :atestado_medico:, :termo_responsabilidade:, :status:)', [...], false)
 5 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Cliente.php(55): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Cliente->create()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
ERROR - 2025-05-08 19:42:02 --> mysqli_sql_exception: Duplicate entry '588.888.888-88' for key 'cliente.CPF' in C:\wamp64\www\sites\Projeto1\Back-end\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `cl...', 0)
#1 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `cl...')
#2 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `cl...')
#3 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `cl...', Array, false)
#4 C:\wamp64\www\sites\Projeto1\Back-end\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\wamp64\www\sites\Projeto1\Back-end\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#6 C:\wamp64\www\sites\Projeto1\Back-end\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\wamp64\www\sites\Projeto1\Back-end\app\Controllers\Cliente.php(55): CodeIgniter\Model->insert(Array)
#8 C:\wamp64\www\sites\Projeto1\Back-end\system\CodeIgniter.php(941): App\Controllers\Cliente->create()
#9 C:\wamp64\www\sites\Projeto1\Back-end\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
#10 C:\wamp64\www\sites\Projeto1\Back-end\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\wamp64\www\sites\Projeto1\Back-end\public\index.php(85): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2025-05-08 19:42:02 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '588.888.888-88' for key 'cliente.CPF'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `cliente` (`nome`, `CPF`, `RG`, `telefone`, `email`, `endereco`, `datanascimento`, `treino_com_personal`, `personal_id`, `atestado_medico`, `termo_responsabilidade`, `status`) VALUES (:nome:, :CPF:, :RG:, :telefone:, :email:, :endereco:, :datanascimento:, :treino_com_personal:, :personal_id:, :atestado_medico:, :termo_responsabilidade:, :status:)', [...], false)
 2 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Cliente.php(55): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Cliente->create()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-05-08 19:42:02 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '588.888.888-88' for key 'cliente.CPF'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `cliente` (`nome`, `CPF`, `RG`, `telefone`, `email`, `endereco`, `datanascimento`, `treino_com_personal`, `personal_id`, `atestado_medico`, `termo_responsabilidade`, `status`) VALUES (\'breno teste\', \'588.888.888-88\', \'48.888.888-8\', \'(12) 99656-9274\', \'breno@breno.com\', \'Av. Macedo Soares, Campos do Jordão\', \'2004-06-07\', \'nao\', \'null\', NULL, NULL, \'inativo\')')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `cliente` (`nome`, `CPF`, `RG`, `telefone`, `email`, `endereco`, `datanascimento`, `treino_com_personal`, `personal_id`, `atestado_medico`, `termo_responsabilidade`, `status`) VALUES (\'breno teste\', \'588.888.888-88\', \'48.888.888-8\', \'(12) 99656-9274\', \'breno@breno.com\', \'Av. Macedo Soares, Campos do Jordão\', \'2004-06-07\', \'nao\', \'null\', NULL, NULL, \'inativo\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `cliente` (`nome`, `CPF`, `RG`, `telefone`, `email`, `endereco`, `datanascimento`, `treino_com_personal`, `personal_id`, `atestado_medico`, `termo_responsabilidade`, `status`) VALUES (:nome:, :CPF:, :RG:, :telefone:, :email:, :endereco:, :datanascimento:, :treino_com_personal:, :personal_id:, :atestado_medico:, :termo_responsabilidade:, :status:)', [...], false)
 4 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\Cliente.php(55): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Cliente->create()
 9 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-05-08 19:42:02 --> [Caused by] mysqli_sql_exception: Duplicate entry '588.888.888-88' for key 'cliente.CPF'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `cliente` (`nome`, `CPF`, `RG`, `telefone`, `email`, `endereco`, `datanascimento`, `treino_com_personal`, `personal_id`, `atestado_medico`, `termo_responsabilidade`, `status`) VALUES (\'breno teste\', \'588.888.888-88\', \'48.888.888-8\', \'(12) 99656-9274\', \'breno@breno.com\', \'Av. Macedo Soares, Campos do Jordão\', \'2004-06-07\', \'nao\', \'null\', NULL, NULL, \'inativo\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `cliente` (`nome`, `CPF`, `RG`, `telefone`, `email`, `endereco`, `datanascimento`, `treino_com_personal`, `personal_id`, `atestado_medico`, `termo_responsabilidade`, `status`) VALUES (\'breno teste\', \'588.888.888-88\', \'48.888.888-8\', \'(12) 99656-9274\', \'breno@breno.com\', \'Av. Macedo Soares, Campos do Jordão\', \'2004-06-07\', \'nao\', \'null\', NULL, NULL, \'inativo\')')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `cliente` (`nome`, `CPF`, `RG`, `telefone`, `email`, `endereco`, `datanascimento`, `treino_com_personal`, `personal_id`, `atestado_medico`, `termo_responsabilidade`, `status`) VALUES (\'breno teste\', \'588.888.888-88\', \'48.888.888-8\', \'(12) 99656-9274\', \'breno@breno.com\', \'Av. Macedo Soares, Campos do Jordão\', \'2004-06-07\', \'nao\', \'null\', NULL, NULL, \'inativo\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `cliente` (`nome`, `CPF`, `RG`, `telefone`, `email`, `endereco`, `datanascimento`, `treino_com_personal`, `personal_id`, `atestado_medico`, `termo_responsabilidade`, `status`) VALUES (:nome:, :CPF:, :RG:, :telefone:, :email:, :endereco:, :datanascimento:, :treino_com_personal:, :personal_id:, :atestado_medico:, :termo_responsabilidade:, :status:)', [...], false)
 5 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Cliente.php(55): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Cliente->create()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
ERROR - 2025-05-08 19:42:13 --> mysqli_sql_exception: Duplicate entry '588.888.888-88' for key 'cliente.CPF' in C:\wamp64\www\sites\Projeto1\Back-end\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `cl...', 0)
#1 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `cl...')
#2 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `cl...')
#3 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `cl...', Array, false)
#4 C:\wamp64\www\sites\Projeto1\Back-end\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\wamp64\www\sites\Projeto1\Back-end\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#6 C:\wamp64\www\sites\Projeto1\Back-end\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\wamp64\www\sites\Projeto1\Back-end\app\Controllers\Cliente.php(55): CodeIgniter\Model->insert(Array)
#8 C:\wamp64\www\sites\Projeto1\Back-end\system\CodeIgniter.php(941): App\Controllers\Cliente->create()
#9 C:\wamp64\www\sites\Projeto1\Back-end\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
#10 C:\wamp64\www\sites\Projeto1\Back-end\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\wamp64\www\sites\Projeto1\Back-end\public\index.php(85): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2025-05-08 19:42:13 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '588.888.888-88' for key 'cliente.CPF'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `cliente` (`nome`, `CPF`, `RG`, `telefone`, `email`, `endereco`, `datanascimento`, `treino_com_personal`, `personal_id`, `atestado_medico`, `termo_responsabilidade`, `status`) VALUES (:nome:, :CPF:, :RG:, :telefone:, :email:, :endereco:, :datanascimento:, :treino_com_personal:, :personal_id:, :atestado_medico:, :termo_responsabilidade:, :status:)', [...], false)
 2 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Cliente.php(55): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Cliente->create()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-05-08 19:42:13 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '588.888.888-88' for key 'cliente.CPF'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `cliente` (`nome`, `CPF`, `RG`, `telefone`, `email`, `endereco`, `datanascimento`, `treino_com_personal`, `personal_id`, `atestado_medico`, `termo_responsabilidade`, `status`) VALUES (\'breno teste\', \'588.888.888-88\', \'48.888.888-8\', \'(12) 99656-9274\', \'breno@breno.com\', \'Av. Macedo Soares, Campos do Jordão\', \'2004-06-07\', \'nao\', \'null\', NULL, NULL, \'inativo\')')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `cliente` (`nome`, `CPF`, `RG`, `telefone`, `email`, `endereco`, `datanascimento`, `treino_com_personal`, `personal_id`, `atestado_medico`, `termo_responsabilidade`, `status`) VALUES (\'breno teste\', \'588.888.888-88\', \'48.888.888-8\', \'(12) 99656-9274\', \'breno@breno.com\', \'Av. Macedo Soares, Campos do Jordão\', \'2004-06-07\', \'nao\', \'null\', NULL, NULL, \'inativo\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `cliente` (`nome`, `CPF`, `RG`, `telefone`, `email`, `endereco`, `datanascimento`, `treino_com_personal`, `personal_id`, `atestado_medico`, `termo_responsabilidade`, `status`) VALUES (:nome:, :CPF:, :RG:, :telefone:, :email:, :endereco:, :datanascimento:, :treino_com_personal:, :personal_id:, :atestado_medico:, :termo_responsabilidade:, :status:)', [...], false)
 4 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\Cliente.php(55): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Cliente->create()
 9 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-05-08 19:42:13 --> [Caused by] mysqli_sql_exception: Duplicate entry '588.888.888-88' for key 'cliente.CPF'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `cliente` (`nome`, `CPF`, `RG`, `telefone`, `email`, `endereco`, `datanascimento`, `treino_com_personal`, `personal_id`, `atestado_medico`, `termo_responsabilidade`, `status`) VALUES (\'breno teste\', \'588.888.888-88\', \'48.888.888-8\', \'(12) 99656-9274\', \'breno@breno.com\', \'Av. Macedo Soares, Campos do Jordão\', \'2004-06-07\', \'nao\', \'null\', NULL, NULL, \'inativo\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `cliente` (`nome`, `CPF`, `RG`, `telefone`, `email`, `endereco`, `datanascimento`, `treino_com_personal`, `personal_id`, `atestado_medico`, `termo_responsabilidade`, `status`) VALUES (\'breno teste\', \'588.888.888-88\', \'48.888.888-8\', \'(12) 99656-9274\', \'breno@breno.com\', \'Av. Macedo Soares, Campos do Jordão\', \'2004-06-07\', \'nao\', \'null\', NULL, NULL, \'inativo\')')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `cliente` (`nome`, `CPF`, `RG`, `telefone`, `email`, `endereco`, `datanascimento`, `treino_com_personal`, `personal_id`, `atestado_medico`, `termo_responsabilidade`, `status`) VALUES (\'breno teste\', \'588.888.888-88\', \'48.888.888-8\', \'(12) 99656-9274\', \'breno@breno.com\', \'Av. Macedo Soares, Campos do Jordão\', \'2004-06-07\', \'nao\', \'null\', NULL, NULL, \'inativo\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `cliente` (`nome`, `CPF`, `RG`, `telefone`, `email`, `endereco`, `datanascimento`, `treino_com_personal`, `personal_id`, `atestado_medico`, `termo_responsabilidade`, `status`) VALUES (:nome:, :CPF:, :RG:, :telefone:, :email:, :endereco:, :datanascimento:, :treino_com_personal:, :personal_id:, :atestado_medico:, :termo_responsabilidade:, :status:)', [...], false)
 5 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Cliente.php(55): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Cliente->create()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
